src/engraving/types/dimension.h

```diff
@@ -106,40 +106,40 @@ class Milimetre
     qreal m_val = 0.0;
 };
 
-//inline Milimetre operator+(const Milimetre& a, const Milimetre& b)
-//{
-//    Milimetre r(a);
-//    r += b;
-//    return r;
-//}
-
-//inline Milimetre operator-(const Milimetre& a, const Milimetre& b)
-//{
-//    Milimetre r(a);
-//    r -= b;
-//    return r;
-//}
-
-//inline Milimetre operator/(const Milimetre& a, qreal b)
-//{
-//    Milimetre r(a);
-//    r /= b;
-//    return r;
-//}
-
-//inline Milimetre operator*(const Milimetre& a, qreal b)
-//{
-//    Milimetre r(a);
-//    r *= b;
-//    return r;
-//}
-
-//inline Milimetre operator*(qreal a, const Milimetre& b)
-//{
-//    Milimetre r(b);
-//    r *= a;
-//    return r;
-//}
+inline Milimetre operator+(const Milimetre& a, const Milimetre& b)
+{
+    Milimetre r(a);
+    r += b;
+    return r;
+}
+
+inline Milimetre operator-(const Milimetre& a, const Milimetre& b)
+{
+    Milimetre r(a);
+    r -= b;
+    return r;
+}
+
+inline Milimetre operator/(const Milimetre& a, qreal b)
+{
+    Milimetre r(a);
+    r /= b;
+    return r;
+}
+
+inline Milimetre operator*(const Milimetre& a, qreal b)
+{
+    Milimetre r(a);
+    r *= b;
+    return r;
+}
+
+inline Milimetre operator*(qreal a, const Milimetre& b)
+{
+    Milimetre r(b);
+    r *= a;
+    return r;
+}
```

Comment:
@igorkorsukov Hi, may I ask why these operator functions were commented out? In this PR I needed to multiply a `Milimetre` with `qreal` so I brought these back to life, and I'm just wondering if there may be side effects.

Reply:
I wanted to know if they were needed, so I commented out (I saw they were not needed and wanted to delete). If you need them, you can uncomment.
I just fear that when there are many such operators, then we can do whatever we want, however we want ... and in the code it may not be clear what is actually happening and how it all works.

Reply:
Thanks for explaining. I am also curious as to why some of the operators aren't defined in the `Milimetre` class itself, like the first four that have a `Milimetre` variable as the first parameter. Is there some kind of tradition concerning what should be defined in the class and what should not be?

Reply:
I believe that the operators with the first parameter are the target class, should be defined in the class declaration.
With one exception, there is no way to change the source code. For example, we cannot change the source code of QString, but we can add an operator outside the declaration of the QString class.
(at the moment these are just copy-paste from Spatium, I was planning to cleaning them)

Reply:
Yes but couldn't we change for example
```
inline Milimetre operator+(const Milimetre& a, const Milimetre& b)
{
    Milimetre r(a);
    r += b;
    return r;
}
```
to
```
inline Milimetre Milimetre::operator+(const Milimetre& b)
{
    Milimetre r(*this);
    r += b;
    return r;
}
```
?

Reply:
Yes, we can change to

Reply:
Got it :)